packages/core/stories/semantic-icon-provider/semantic-icon-provider.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from "@storybook/addon-docs";
+import { Banner, BannerContent } from "@salt-ds/core";
+
+<Meta title="Experimental/Semantic Icon Provider" />
```

Comment:
Are we considering this experimental? If so, needs to mark unstable?

Reply:
moved it to core